packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -1,4 +1,11 @@
 .font-library-modal {
+	&.is-full-screen {
```

Comment:
Is there a way we can avoid using `is-full-screen` ? It is an internal classname used by the component and therefore it shouldn't be used by its consumers.

Reply:
This selector is required for increased CSS specificity. However, as I mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/54518#discussion_r1328424969), we can remove this CSS by using the `useViewportMatch` hook instead. What do you think about the hook approach?

Or, I think it can be solved by duplicating the component selectors:

```css
.font-library-modal {
	&.font-library-modal {
		@include break-medium() {
			width: 65vw;
		}
	}
}
```